src/openforms/config/models/config.py

```diff
@@ -321,15 +321,18 @@ class GlobalConfiguration(SingletonModel):
     )
 
     # global payment settings
-    payment_order_id_prefix = models.CharField(
-        _("Payment Order ID prefix"),
-        max_length=16,
-        default="{year}",
-        blank=True,
-        help_text=_(
-            "Prefix to apply to generated numerical order IDs. Alpha-numerical only, supports placeholder {year}."
-        ),
-        validators=[validate_payment_order_id_prefix],
+    payment_order_id_template = models.CharField(
+        _("Payment Order ID template"),
+        max_length=48,
+        default="{year}/{reference}/{uid}",
+        help_text=_(
+            "Template to use when generating payment order IDs. It should be alpha-numerical and can contain the '/._-' characters. "
+            "The following placeholders are supported:\n"
+            "  - {year}: The current year\n"
+            "  - {reference}: The submission reference\n"
+            "  - {uid}: A unique incrementing payment ID."
+        ),
```

Comment:
Newlines don't translate to HTML. Please write your help texts in a more simple form, and add verbose Sphinx documentation to provide more context/background. Your last three lines can be replaced with: "You can use the placeholder tokens: {year}, {public_reference}, {uid}."